### DISCUSSION TITLE
[Bug] In LAN, hotspot host can't access client service, but client can access host service.

### DISCUSSION THREAD

**Discussion by yingshaoxo** [Mygod/VPNHotspot#493] [Q&A]
## Reproduce: 
1. Host open 'repeater'. It has IP of `192.168.49.1/24`.
2. Another phone connect to it, by using [simple http server](https://play.google.com/store/apps/details?id=com.phlox.simpleserver&hl=en_US&gl=US), it got ip address of `192.168.49.32:8080`
3. In host phone, by visiting `http://192.168.49.32:8080`, I got no response.

## But
If I use host phone to start a  [simple http server](https://play.google.com/store/apps/details?id=com.phlox.simpleserver&hl=en_US&gl=US) service
If I use client phone to visit `http://192.168.49.1:8080`, I got response

**Comment by Mygod** [comment]
Double check your VPN configuration.

**Comment by yingshaoxo** [comment]
Thank you, now I solved this problem.

I should add the following to v2ray client config:

```json
"routing": {
    "domainMatcher": "mph",
    "domainStrategy": "AsIs",
    "rules": [
      {
        "ip": [
          "geoip:private"
        ],
        "outboundTag": "direct",
        "type": "field"
      }
    ]
}
```
